.github/workflows/validate.yml

```diff
@@ -148,6 +148,17 @@ jobs:
         make cfmt
         git diff --exit-code
 
+  gosec:
```

Comment:
Give. that we already run golang-ci linter, which also provides gosec linter, would we instead enable it in the golangci linter config?